.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -9,39 +9,39 @@ In order to ease downstream packaging of Rust binaries, an archive of vendored d
 
 This guide requires:
 
- * a web browser (and network connectivity)
+ * A web browser (and network connectivity)
  * `git`
- * GPG setup and personal key for signing
+ * [GPG setup][GPG setup] and personal key for signing git commits
```

Comment:
Really it's the tag that matters more than the commits.  `cargo release` happens to sign the commits but that's not really important.

Reply:
I see. Hmm, I guess it's fine to keep it more general as it was before, then.